src/main/kotlin/blackjack/view/InputView.kt

```diff
@@ -0,0 +1,31 @@
+package blackjack.view
+
+import blackjack.model.ErrorMessage
+import blackjack.model.Player
+
+object InputView {
+    fun readNames(): List<String> {
+        println("\nEnter the names of the players (comma-separated):")
+        val input = readln()
+        if (input.isEmpty()) {
+            throw IllegalArgumentException(ErrorMessage.EMPTY_INPUT.toString())
+        }
+        return input.split(",").map { it.trim() }
+    }
+
+    fun promptForDraw(player: Player): Boolean {
+        println("\nWould ${player.name} like to draw another card? (y for yes, n for no)")
+        val input = readln().trim()
+        if (input.isEmpty()) {
+            throw IllegalArgumentException(ErrorMessage.EMPTY_INPUT.toString())
+        }
```

Comment:
```suggestion
        require(input.isNotEmpty()) { ErrorMessage.EMPTY_INPUT.toString() }
```

We can utilize `require` for this logic!

Reply:
Thanks for the suggestion!